src/Processors/Formats/Impl/JSONCompactEachRowRowOutputFormat.cpp

```diff
@@ -55,9 +55,8 @@ void JSONCompactEachRowRowOutputFormat::writeRowEndDelimiter()
 void JSONCompactEachRowRowOutputFormat::writeTotals(const Columns & columns, size_t row_num)
 {
     writeChar('\n', out);
-    size_t num_columns = columns.size();
     writeRowStartDelimiter();
-    for (size_t i = 0; i < num_columns; ++i)
+    for (size_t i = 0; i < columns.size(); ++i)
```

Comment:
Actually this is not the same as was before, because compiler generate different code if size is used as loop predicate, even if method is visible on compile time.

Reply:
I modified it because it would cause the compile warning:
```
/home/code/github/clickhouse_github/src/Processors/Formats/Impl/JSONCompactEachRowRowOutputFormat.cpp:58:12: error: declaration of ‘num_columns’ shadows a member of ‘DB::JSONCompactEachRowRowOutputFormat’ [-Werror=shadow]
   58 |     size_t num_columns = columns.size();
```
Do you mean that the former has better performance?

Reply:
Yes, we can just rename it into `columns_size`.

Reply:
done